### PR TITLE
Add location of where to install packages

### PR DESCRIPTION
**Pull request by veteranmina**
Clarify the file location and variable to install packages

### CODE REVIEW

File: docs/community_packages.md [hunk 1]
```diff
@@ -4,13 +4,17 @@
 
 Below is a list of packages contributed by the community. These packages normally follow the same installation procedure, however, its best you consult the documentation of the package itself in case there is anything special you need to get it working.
 
+Applying community packages to SeAT requires modifying your .env file located at /var/www/seat/.env for bare metal installs and /opt/seat-docker/.env for docker installs. To add the package(s) you must uncomment SEAT_PLUGINS by removing the '#' and entering the package(s) to be installed following the guidelines provided.
```
Comment (leonjza) [inline docs/community_packages.md:7]: Bare metal installs don't have the same `.env` file as docker installs. They have to manually do the composer commands that the docker install will do for you.

Comment (veteranmina) [reply docs/community_packages.md]: I've updated the document removing the bare metal references I added and cleaned up the Docker instructions per my last commit.

**Comment by warlof** [comment]
Maybe you can add a `.env` file sample so people got an idea of how it should looks ?
Preferably containing multiple packages (so, using `,` comma separator) - out of that, sounds good :)

**Comment by warlof** [comment]
thank you very much for this 👍

**Pull request merged**